src/releasers/java/version.ts

```diff
@@ -94,15 +94,22 @@ export class Version {
         break;
       case 'lts':
         if (this.lts) {
-          this.lts += 1;
-        } else {
           if (!this.snapshot) {
```

Comment:
mind adding a couple comments to `snapshot` vs., `lts-snapshot` explaining the nuances between how we bump the two types of releases?